CHANGES.rst

```diff
@@ -4,11 +4,13 @@ Changes by Version
 0.19.0 (unreleased)
 -------------------
 
-- Reduced Zipkin submission failures to warnings.
-- **BREAKING** - removed ``tchannel.thrift.client_for`` and
+- **BREAKING** - Removed Apache Thrift support. The ``thrift`` module is no
+  longer necessary.
```

Comment:
What ``thrift`` module? We still have the `tchannel.thrift` module for `thrift.load`.